### PR TITLE
Oadp 1488 oadp 1 1 4 release notes hvider

### PR DESCRIPTION
**Pull request by HagayVider1**
@anarnold97

OCPBUGS# : [OADP-1488](https://issues.redhat.com//browse/OADP-1488)
OCPBUGS# : [OADP-1488](https://issues.redhat.com//browse/OADP-1488)

Version(s): OADP 1.1.4

Issue: https://issues.redhat.com/browse/OADP-1488

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/oadp-release-notes-1-1-4.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+[id="resolved-issues_{context}"]
+== Resolved issues
+
+The issues and bugs that have been resolved in this version are listed in link:https://issues.redhat.com/browse/OADP-1398?jql=project%20%3D%20OADP%20AND%20status%20in%20(Closed%2C%20Verified%2C%20%22Release%20Pending%22)%20AND%20priority%20in%20(Blocker%2C%20Critical%2C%20Major)%20AND%20fixVersion%20in%20(%22OADP%201.1.4%22%2C%20%22OADP%201.1.4%22)%20AND%20assignee%20not%20in%20(rhn-support-hvider%2C%20rhn-support-anarnold%2C%20rhn-support-cwisemon%2C%20richard.hoch%2C%20rhn-support-sbeskin)%20ORDER%20BY%20description%20ASC[this page].
+
+[id="known-issues_{context}"]
+== Known issues
+
+The issues and bugs that are known in this version are listed in link:https://issues.redhat.com/browse/OADP-1619?jql=project%20%3D%20OADP%20AND%20status%20not%20in%20(Closed%2C%20Verified%2C%20%22Release%20Pending%22)%20AND%20priority%20in%20(Blocker%2C%20Critical%2C%20Major)%20AND%20fixVersion%20in%20(%22OADP%201.1.4%22%2C%20%22OADP%201.1.4%22)%20AND%20assignee%20not%20in%20(rhn-support-hvider%2C%20rhn-support-anarnold%2C%20rhn-support-cwisemon%2C%20richard.hoch%2C%20rhn-support-sbeskin)%20ORDER%20BY%20description%20ASC[this page].
```
Comment (weshayutin) [inline modules/oadp-release-notes-1-1-4.adoc:24]: why would we include the release task here? @rayfordj wdyt..  is that needed?

Comment (anarnold97) [reply modules/oadp-release-notes-1-1-4.adoc:24]: @weshayutin @rayfordj

Apologies, I will come back and do a re-write later tonight. There are also build errors which we need to fix.

Comment (rayfordj) [reply modules/oadp-release-notes-1-1-4.adoc:24]: Thanks! :sunglasses:

**Comment by anarnold97** [comment]
@HagayVider1 - I re-did the release notes - https://github.com/openshift/openshift-docs/pull/60136
You have copied the style of MTA and not use that of [OADP for the release notes](https://docs.openshift.com/container-platform/4.11/backup_and_restore/application_backup_and_restore/oadp-release-notes.html#migration-oadp-release-notes-1-1-1_oadp-release-notes)
Also, there are build errors being returned by Travis.
The query for  you are using are not great either.

**Pull request closed without merging**